.gitlab-ci.yml

```
@@ -131,15 +131,14 @@ publish:image-multiplatform:mender:worker:
 
 publish:acceptance:
   stage: publish
-  except:
-    - /^saas-[a-zA-Z0-9.]+$/
-  image: golang:1.14-alpine3.11
+  rules:
+    - if: $CI_COMMIT_BRANCH =~ /^saas-[a-zA-Z0-9.]+$/
+      when: never
+    - when: on_success
```

It is the same as:

```suggestion
  rules:
    - if: $CI_COMMIT_BRANCH !~ /^saas-[a-zA-Z0-9.]+$/
```

`when: on_success` is the default, but if you like it explicit you ca use:
```suggestion
  rules:
    - if: $CI_COMMIT_BRANCH !~ /^saas-[a-zA-Z0-9.]+$/
      when: on_success
```

---

Right! Thanks for the hints!